_sass/themes/_light.scss

```diff
@@ -9,7 +9,7 @@ $backgroundColor: $white;
 $textColor: #535f61;
 $textCodeColor: #d14;
 
-$linkColor: #469BFF;
+$linkColor: #469Bdd;
```

Comment:
I'd go a bit darker/further to what it used to be but thanks for going in that direction already !